app-nested-containers/build.gradle

```diff
@@ -33,21 +33,22 @@ android {
         compose true
     }
     composeOptions {
-        kotlinCompilerExtensionVersion "0.1.0-dev09"
+        kotlinCompilerVersion "1.3.70-dev-withExperimentalGoogleExtensions-20200424"
+        kotlinCompilerExtensionVersion "0.1.0-dev12"
```

Comment:
I would suggest to use dev-13, if you are updating this lot
We can remove dependency on router project later in next migrations